vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-functions.xml

```diff
@@ -323,6 +323,13 @@
         <return type="document-node()?"/>
         <runtime type="scalar" class="org.apache.vxquery.runtime.functions.node.FnDocScalarEvaluatorFactory"/>
     </function>
+    
+    <!-- jn:doc($uri as  xs:string?) as document-node()?  -->
+    <function name="fn:doc-json">
```

Comment:
Should be called jn:json-doc [1]

[1] http://jsoniq.org/docs/JSONiqExtensionToXQuery/html-single/index.html#idm139680622843744